<?xml version="1.0" encoding="utf-8"?>
<launch>

  <param name ="/use_sim_time" value="false"/>
  <node  pkg="cam_lidar_calib" name="cam_lidar_calib" type="cam_lidar_calib_node" output="screen">
  	<param name = "dx" type="double" value="0.108"/>
  	<param name = "dy" type="double" value="0.108"/>

  	<param name = "checkerboard_rows" type = "int" value = "8" />
  	<param name = "checkerboard_cols" type = "int" value = "6" />

  	<param name = "result_file" type="string" value="$(find cam_lidar_calib)/result/C_T_L_logi_VLP_GP.txt"/>
  	<param name = "result_rpy_file" type="string" value="$(find cam_lidar_calib)/result/rpy_txyz_logi.txt"/>

  	<param name = "min_points_on_plane" type="int" value="850"/>
  	<param name = "num_views" type="int" value="19"/>

  	<param name = "camera_in_topic" type="string" value="/usb_cam/image_raw"/>
  	<param name = "lidar_in_topic" type="string" value="/velodyne_points"/>

	<param name="x_min" value="0"/>
    	<param name="x_max" value="6"/>
    	<param name="y_min" value="-1.25"/>
    	<param name="y_max" value="1.25"/>
    	<param name="z_min" value="-0.5"/>
    	<param name="z_max" value="2"/>
    	<param name="ransac_threshold" value="0.01"/>

    <param name="no_of_initializations" value="1" />
    <param name="initializations_file" value="/home/frozen/catkin_ws/src/cam_lidar_calib/result/logi_result.txt" />


  	<param name = "cam_config_file_path" type="string" value="$(find cam_lidar_calib)/config/logi.yaml" />
  </node>
</launch>
